diff --git a/components/esp_lcd/parl/esp_lcd_panel_io_parl.c b/components/esp_lcd/parl/esp_lcd_panel_io_parl.c
index efdba5926af..1c22adaa39f 100644
--- a/components/esp_lcd/parl/esp_lcd_panel_io_parl.c
+++ b/components/esp_lcd/parl/esp_lcd_panel_io_parl.c
@@ -115,7 +115,7 @@ esp_err_t esp_lcd_new_panel_io_parl(const esp_lcd_panel_io_parl_config_t *io_con
         .output_clk_freq_hz = io_config->pclk_hz,
         .trans_queue_depth = io_config->trans_queue_depth ? io_config->trans_queue_depth : 4,
         .max_transfer_size = io_config->max_transfer_bytes,
-        .sample_edge = PARLIO_SAMPLE_EDGE_POS,
+        .sample_edge = PARLIO_SAMPLE_EDGE_NEG,
         .bit_pack_order = PARLIO_BIT_PACK_ORDER_MSB,
         .dma_burst_size = io_config->dma_burst_size,
         .flags.invert_valid_out = !io_config->flags.cs_active_high,
diff --git a/components/usb/include/usb/usb_types_ch9.h b/components/usb/include/usb/usb_types_ch9.h
index f54f94842a0..a1214324fd2 100644
--- a/components/usb/include/usb/usb_types_ch9.h
+++ b/components/usb/include/usb/usb_types_ch9.h
@@ -384,7 +384,7 @@ ESP_STATIC_ASSERT(sizeof(usb_config_desc_t) == USB_CONFIG_DESC_SIZE, "Size of us
 /**
  * @brief Size of a USB interface association descriptor in bytes
  */
-#define USB_IAD_DESC_SIZE    9
+#define USB_IAD_DESC_SIZE    8
 
 /**
  * @brief Structure representing a USB interface association descriptor
